### PR TITLE
Wall spell

### PR DESCRIPTION
**Pull request by OwenBoisse**
creates wall behind you as you walk

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -16,6 +16,7 @@ use bevy::diagnostic::{FrameTimeDiagnosticsPlugin, LogDiagnosticsPlugin};
 use bevy::prelude::*;
 use bevy::transform::components::Transform;
 use bevy_ggrs::GgrsConfig;
+use bevy_hanabi::HanabiPlugin;
```
Comment (AmosHolland) [inline src/lib.rs:19]: What's this doing here?

**Review by AmosHolland — CHANGES_REQUESTED** [review]
One minor nitpick, otherwise this all looks good!

File: src/lib.rs (hunk 1 above)
Comment (OwenBoisse) [reply src/lib.rs:19]: I may have accidentally deleted the import in the effect asset branch when it was pulled into main so its missing the import for hanabi and won't run without it (unless you remove all the hanabi stuff which ig I could do in this branch)

**Review by AmosHolland — APPROVED** [review]
Send it!

**Review by rosefromthedead — APPROVED** [review]

**Pull request merged**